src/test/java/com/spotify/docker/client/DockerHostTest.java

```diff
@@ -98,7 +98,7 @@ public void testCertPathFromEnv() throws Exception {
     DockerHost.setSystemDelegate(systemDelegate);
 
     assertThat(DockerHost.certPathFromEnv(), equalTo("foo"));
-    assertThat(DockerHost.certPathFromEnv(), equalTo("bar/.docker"));
+    assertThat(DockerHost.certPathFromEnv(), equalTo(null));
```

Comment:
Mockito lets you specify different values to return in sequence when you hit a certain method. You can also tell it to throw exceptions. It's pretty nice.

The point is so you can test different paths of a conditional. You’re testing "When this method (not under test) returns `foo`, I expect my code under test to do a, b, and c, but when it returns `null` I expect x, y, and z to happen."